eks/bastion.tf

```diff
@@ -1,18 +1,46 @@
 resource "aws_eip" "bastion_eip" {
   instance = aws_instance.bastion.id
-  vpc      = true
+  vpc = true
 }
 
 resource "aws_instance" "bastion" {
-  ami           = data.aws_ami.opszero_eks.id
+  //  ami = data.aws_ami.opszero_eks.id
+  ami = "ami-049aea444f70407b8"
```

Comment:
Maybe we just create 2 boxes? One for the VPN and one for the Bastion?

Reply:
Actually lets do that. Create a separate resource which is vpn.

Reply:
Made a new vpn resource